ui.apps/src/main/content/jcr_root/apps/core/cif/components/content/button/v1/button/button.html

```diff
@@ -13,6 +13,6 @@
   ~ See the License for the specific language governing permissions and
   ~ limitations under the License.
   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/-->
-<a data-sly-use.button="com.adobe.cq.wcm.core.components.models.Button" class="button button--secondary" href="${button.link}" role="button">
+<a data-sly-use.button="com.adobe.cq.wcm.core.components.models.Button" class="button button--secondary" href="${button.link}" role="button" title="${button.text}">
```

Comment:
I don't think we should duplicate the buttons label (text content) with the title attribute. According to w3 the title supplements the link text.

> The objective of this technique is to demonstrate how to use a title attribute on an anchor element to provide additional text describing a link. The title attribute is used to provide additional information to help clarify or further describe the purpose of a link. If the supplementary information provided through the title attribute is something the user should know before following the link, such as a warning, then it should be provided in the link text rather than in the title attribute.

https://www.w3.org/TR/WCAG20-TECHS/H33.html

When the Button is a product or category link, can it be the product title or category title respectively? If not I would skip the button for now and create a followup story?

Reply:
Then +1 for skipping it. This is Button V1 and should not be used anyway.

Reply:
I reverted the change to the button component 👍